src/crypto/index.js

```diff
@@ -106,6 +100,20 @@ function Crypto(baseApis, sessionStore, userId, deviceId,
     this._receivedRoomKeyRequestCancellations = [];
     // true if we are currently processing received room key requests
     this._processingRoomKeyRequests = false;
+}
+utils.inherits(Crypto, EventEmitter);
+
+/**
+ * Initialise the crypto module so that it is ready for use
```

Comment:
You should mention that it returns a promise and that it is only initialised once said promise resolves.